### PR TITLE
Questions now fill the input box when clicked

### PR DESCRIPTION
**Pull request by stevenhillcox**

### CODE REVIEW

File: src/client/app/chat-room/chat-room.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <chat-room-info></chat-room-info>
 <div class="middle-bar">
     <chat-room-display></chat-room-display>
-    <chat-room-input></chat-room-input>
+    <chat-room-input #input></chat-room-input>
 </div>
-<chat-room-questions></chat-room-questions>
\ No newline at end of file
+<chat-room-questions (select)="input.setInput($event)"></chat-room-questions>
```
Comment (kenneth-gray) [inline src/client/app/chat-room/chat-room.component.html]: `(questionSelect)`? Just a suggestion - if you're happy with the current name then that works too.

Comment (stevenhillcox) [inline src/client/app/chat-room/chat-room.component.html]: I feel that the question component is quite small and won't really be extended with anything that might cause confusion (i.e. another element to be selected)

File: src/client/app/chat-room/-input/chat-room-input.component.ts [hunk 2]
```diff
@@ -20,4 +20,8 @@ export default class ChatRoomInputComponent {
         }
         this.input = '';
     }
+
+    setInput(text) {
```
Comment (kenneth-gray) [inline src/client/app/chat-room/-input/chat-room-input.component.ts]: Does this need a test?

File: src/client/app/chat-room/chat-room.component.html (hunk 1 above)
Comment (kenneth-gray) [inline src/client/app/chat-room/chat-room.component.html]: Sure

**Pull request merged**